ingest/pipeline/register_test.go

```diff
@@ -0,0 +1,76 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package pipeline
+
+import (
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/elastic/beats/libbeat/common"
+	es "github.com/elastic/beats/libbeat/outputs/elasticsearch"
+)
+
+func TestRegisterPipelines(t *testing.T) {
+	esClients, err := es.NewElasticsearchClients(getFakeESConfig(9200))
+	require.NoError(t, err)
+	esClient := &esClients[0]
+	_, current, _, _ := runtime.Caller(0)
+	path := filepath.Join(filepath.Dir(current), "..", "..", "ingest", "pipeline", "definition.json")
```

Comment:
not important, but i'd try to reuse functions in `tests/loader.go` for these things